bpf/gotracer/go_sql.c

```diff
@@ -114,7 +122,8 @@ read_mysql_hostname_from_mysqlconn(void *mysql_conn_ptr, char *hostname, u64 max
 
 // SQL hostname extraction with driver type routing.
 // Attempts to extract hostname by trying supported database drivers
-static __always_inline void extract_sql_hostname(sql_request_trace_t *trace, u64 driver_conn_ptr) {
+static __always_inline void
+extract_sql_hostname(sql_request_trace_t *trace, u64 driver_conn_ptr, void *goroutine_addr) {
     trace->hostname[0] = '\0';
 
     if (driver_conn_ptr == 0) {
```

Comment:
Could this check prevent us now from fetching the postgres data? it seems that the postgres support doesn't depend on the driver connection pointer. Perhaps best is to move the code for postgres forward ahead of this check.